File: Domains/ApplicationDomain/BOA/Entities/Order.cs
using ApplicationDomain.Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.BOA.Entities
{
    public class Order : EntityBase<int>
    {
        public string Code { get; set; }
        public int EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public int FoodId { get; set; }
        public Food Food { get; set; }
        public int Quantity { get; set; }
        public int FarmerId { get; set; }
        public Farmer Farmer { get; set; }
    }
}

File: Applications/WebAdminApplication/ClientApp/src/app/modules/order/models/order.ts
export class Order {
  id: number;
  code: string;
  employeeId: number;
  foodId: number;
  quantity: number;
  farmerId: number
  employeeName: string;
  farmerName: string;
  foodName: string;

  constructor() {
   this.id =0;
   this.code = "";
   this.employeeId = 0;
   this.foodId = 0;
   this.quantity = 0;
   this.farmerId = 0;
  }
}
